### DISCUSSION TITLE
从v4到v5有较大的改动，根据官方文档的迁移教程不能完成迁移，可以再具体一些吗，主要是想用webgl渲染器加速渲染的能力

### DISCUSSION THREAD

**Discussion by lishanlei** [antvis/G2#5589] [Q&A]
- **G2 Version**:
- **Platform**:
- **Mini Showcase(like screenshots)**:
- **CodePen Link**:

**Comment by pearmini** [comment]
是希望什么部分具体法？或者希望迁移文档怎么写更好？

**Comment by lishanlei** [comment]
我在项目里用了比较多的v4里导出的方法、类以及interface，例如类Chart，View，Geometry等，以及v4里导出的lib下面的interface里的很多type，但是在v5里很难对应的上。当然，也有可能是我对新版G2理解的还不够透彻，不过还是希望能予以解答，谢谢！

**Comment by pearmini** [comment]
你可以把你难以理解的 type 列举一下，然后我一一回复？
